From 4c08f9480bbb5eee7a4d1f69652acee8f5204a46 Mon Sep 17 00:00:00 2001
From: Julius Kammerl <jkammerl@rbh.willowgarage.com>
Date: Wed, 5 Dec 2012 19:27:48 -0800
Subject: [PATCH] class_loader_hide_library_symbols

---
 CMakeLists.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59e1eb4..b22688b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,11 +40,14 @@ target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES}
 #                                      ogg theora theoraenc theoradec
 )
 
+class_loader_hide_library_symbols(${PROJECT_NAME})
+
 add_executable(ogg_saver src/ogg_saver.cpp)
 include_directories(ogg_saver include cfg/cpp)
 target_link_libraries(ogg_saver ${PC_THEORA_LIBRARY} ${PC_OGG_LIBRARY} ${OpenCV_LIBRARIES} ${catkin_LIBRARIES} ogg theora theoraenc theoradec)
 add_dependencies(ogg_saver ${PROJECT_NAME}_gencpp)
 
+class_loader_hide_library_symbols(ogg_saver)
 
 install(TARGETS ${PROJECT_NAME}
         DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-- 
1.7.9.5

